dracut/modules.d/90kiwi-dump/kiwi-dump-image.sh

```diff
@@ -45,10 +61,16 @@ function get_disk_list {
     local disk_device_by_id
     local disk_meta
     local list_items
+    local max_disk md_string
```

Comment:
This is nitpick, can we specify one variable per line

```
local max_disk
local md_string
```

That having said I wouldn't know that md_string will contain the contents of the rd.kiwi.oem.maxdisk option. Would you consider a variable name that makes this more clear. e.g

```
kiwi_oem_maxdisk=(getarg rd.kiwi.oem.maxdisk=)
```

Reply:
OK (to all comments ;) )